vignettes/baselinenowcast.Rmd

```diff
@@ -187,7 +191,7 @@ n_training_volume <- 3 * max_delay
 n_history_delay <- 0.5 * n_training_volume
 
 # Specify the number of reference times to use for uncertainty estimation.
-n_history_uncertainty <- 0.5 * n_training_volume
+n_ref_times_for_uncertainty <- 0.5 * n_training_volume
```

Comment:
why does this one refer to ref_times and the delay one doesn't? 

I think n_retrospective_nowcasts would be a clear name tbh.